Dockerfile.test

```diff
@@ -16,18 +18,21 @@ RUN wget http://python.org/ftp/python/$pyVer/Python-${pyVer}.tgz && \
 	make && \
 	checkinstall
 
-RUN apt-get install -y python-dev python-pip
-RUN pip install psutil && \
-    pip install netifaces && \
-	pip install bottle && \
-	pip install requests && \
-	pip install python-dateutil
-
 RUN mkdir /crawler
 COPY . /crawler/
 WORKDIR /crawler/
 
-RUN pip2 install --no-index --find-links="packaging/python_packages/" psutil bottle requests pydoubles netifaces kafka-python docker-py pykafka
+RUN apt-get install -y python-dev python-pip
+RUN pip install --upgrade pip
+RUN pip install -r crawler/requirements.txt
+
+# tests are pointing to this directory
+RUN ln -s crawler config_and_metrics_crawler
 
 WORKDIR tests
-CMD (docker daemon > ../docker.out 2>&1 &) && sleep 5 && ./test_all.sh
+# let's pull an ubuntu image before starting the tests
+CMD (docker daemon > ../docker.out 2>&1 &) && \
+	sleep 5 && \
+	docker pull ubuntu && \
```

Comment:
That image is needed for some of the tests. It's going to be pulled by one of the tests anyway. It's just that doing it as part of the test might make the test fail. Yet another reason of why these tests need some work.